internal/operator/controller/contour/controller.go

```diff
@@ -63,7 +63,7 @@ type reconciler struct {
 // +kubebuilder:rbac:groups="",resources=namespaces;secrets;serviceaccounts;services,verbs=get;list;watch;delete;create;update
 // +kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch;delete;create;update
 // +kubebuilder:rbac:groups="",resources=endpoints,verbs=get;list;watch
-// +kubebuilder:rbac:groups=networking.k8s.io,resources=gatewayclasses;gateways;httproutes;tcproutes;ingresses;backendpolicies,verbs=get;list;watch
+// +kubebuilder:rbac:groups=networking.k8s.io,resources=gatewayclasses;gateways;httproutes;tcproutes;ingresses;ingressclasses;backendpolicies,verbs=get;list;watch
```

Comment:
As an aside, should the service-apis group be `networking.x-k8s.io` and not `networking.k8s.io`? This is what I have in Contour which puts the `ingresses` & `ingressclasses` into different groups.

Reply:
The apigroup is definitely `networking.x-k8s.io` in the upstream code for the service-apis, but should be `networking.k8s.io` for all the Ingress and IngressClass stuff.

Reply:
ack